docs/developers/wallet.md

```diff
@@ -27,7 +27,7 @@ celestia-appd keys list
 ## Fund a Wallet
 
 For the public celestia address, you can fund the
-previously created wallet via Discord by sending
+previously created wallet via [Discord](https://discord.gg/celestiacommunity) by sending
```

Comment:
Can you fix the lint errors?

You can do that by moving "by sending" to the next line so to not exceed 80 chars per line. 

Also make sure there's no blank space at the end of each line you added.

Reply:
Thank you for the feedback, I just realized after reading this comment that I didn't have the markdownlint extension installed on VSCode! Just installed it, hope all is good now!

Here is the new PR following the steps outlined in the comment!:
https://github.com/celestiaorg/docs/pull/195/commits/707aa91e094284d0aeae838c99f8402963a47961